src/GoogleAnalytics.svelte

```diff
@@ -32,7 +33,8 @@
     window.dataLayer = window.dataLayer || []
     gtag('js', new Date())
     properties.forEach(p => {
-      gtag('config', p)
+      const propertyConfig = configurations && configurations[p] || undefined
```

Comment:
isnt `configurations[p]` undefined anyway if the entry doesn't exist? :) You could probably save a line of code here.

Reply:
Yeah, not sure why I did that extra type check on `configurations` since it has a default value.
👍